sunpy/net/jsoc/jsoc.py

```diff
@@ -213,13 +221,26 @@ def query(self, *query, **kwargs):
             iargs = kwargs.copy()
             iargs.update(block)
             blocks.append(iargs)
-
             return_results.append(self._lookup_records(iargs))
 
         return_results.query_args = blocks
-
         return return_results
 
+    def get_metadata(self, *query, **kwargs):
+
+        query = and_(*query)
+        blocks = []
+        dicts = {}
+        for block in walker.create(query):
+            iargs = kwargs.copy()
+            iargs.update(block)
+            iargs.update({'meta': True})
+            blocks.append(iargs)
+            r = self._lookup_records(iargs)
```

Comment:
Well, its type is a DataFrame. What do you suggest? I converted it to a dict, because I saw this object MetaDict which stores metadata of FITS files. And that's why, I converted it into MetaDict object. What do you say? Should I keep it a MetaDict or DataFrame or Astropy table?